pkg/volume/util/operationexecutor/operation_generator_test.go

```diff
@@ -166,6 +167,150 @@ func TestOperationGenerator_GenerateUnmapVolumeFunc_PluginName(t *testing.T) {
+			},
+			probVolumePlugins: awsebs.ProbeVolumePlugins(),
+		},
+		{
+			name:                  "aws plugin: csi migration disabled - fs volume",
+			isCsiMigrationEnabled: false,
+			callOnlineExpansion:   true,
+			pluginName:            plugins.AWSEBSInTreePluginName,
+			csiMigrationFeature:   features.CSIMigrationAWS,
+			pvSpec: v1.PersistentVolumeSpec{
+				PersistentVolumeSource: v1.PersistentVolumeSource{
+					AWSElasticBlockStore: &v1.AWSElasticBlockStoreVolumeSource{
+						VolumeID: "vol-1234",
+					},
+				},
+				VolumeMode: &fsMode,
+			},
+			probVolumePlugins: awsebs.ProbeVolumePlugins(),
+		},
+		{
+			name:                  "regular csi plugin - block volume",
+			isCsiMigrationEnabled: false,
+			callOnlineExpansion:   true,
+			pluginName:            "kubernetes.io/csi",
+			csiMigrationFeature:   features.CSIMigration,
+			pvSpec: v1.PersistentVolumeSpec{
+				PersistentVolumeSource: v1.PersistentVolumeSource{
+					CSI: &v1.CSIPersistentVolumeSource{
+						Driver:       plugins.AWSEBSDriverName,
+						VolumeHandle: "vol-xyze",
+					},
+				},
+				VolumeMode: &blockMode,
+			},
+			probVolumePlugins: []volume.VolumePlugin{},
+		},
+		{
+			name:                  "regular csi plugin - fs volume",
+			isCsiMigrationEnabled: false,
+			callOnlineExpansion:   true,
+			pluginName:            "kubernetes.io/csi",
+			csiMigrationFeature:   features.CSIMigration,
+			pvSpec: v1.PersistentVolumeSpec{
+				PersistentVolumeSource: v1.PersistentVolumeSource{
+					CSI: &v1.CSIPersistentVolumeSource{
+						Driver:       plugins.AWSEBSDriverName,
+						VolumeHandle: "vol-xyze",
+					},
+				},
+				VolumeMode: &fsMode,
+			},
+			probVolumePlugins: []volume.VolumePlugin{},
+		},
+	}
+
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIMigration, tc.isCsiMigrationEnabled)()
+			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, tc.csiMigrationFeature, tc.isCsiMigrationEnabled)()
+
+			volumePluginMgr, _, tmpDir := initTestPlugins(t, tc.probVolumePlugins, tc.pluginName)
+			defer os.RemoveAll(tmpDir)
+
+			ogInterface := getTestOperationGenerator(volumePluginMgr)
+			onlineExpansionCalled := false
+
+			og, _ := ogInterface.(*operationGenerator)
+			og.onlineExpansionFunc = func(_ VolumeToMount, _ ActualStateOfWorldMounterUpdater, _ volume.NodeResizeOptions) (bool, error, error) {
+				onlineExpansionCalled = true
+				return true, nil, nil
+			}
+
+			pod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{UID: types.UID(string(uuid.NewUUID()))}}
+			volumeToExpand := getTestVolumeToMount(pod, tc.pvSpec, tc.pluginName)
+
+			expandVolumeFunc, e := ogInterface.GenerateExpandInUseVolumeFunc(volumeToExpand, nil)
+			if e != nil {
+				t.Fatalf("Error occurred while generating expandInUseVolumeFunc: %v", e)
+			}
+
+			e1, e2 := expandVolumeFunc.OperationFunc()
+			if e1 != nil {
```

Comment:
This could be written as one line:

```golang
if e1 != nil || e2 != nil {
    t.Fatalf("unable to call volume expansion: %v, %v", e1, e2)
}
```